go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/containers/podman/v4
 
 go 1.18
 
+replace github.com/containers/common v0.53.1-0.20230506101404-3e93a76d461c => github.com/Cydox/common v0.0.0-20230523224052-80eac653d332
```

Comment:
You'll need to get your changes from your fork into the upstream branch and vendor from there before merging this.

Reply:
I would just revendor in your sandbox, using the latest upstream c/common with your change in it, then repush this PR.